cuda_core/tests/test_program.py

```diff
@@ -176,6 +193,13 @@ def ptx_code_object():
     [
         ProgramOptions(name="abc"),
         ProgramOptions(device_code_optimize=True, debug=True),
+        pytest.param(
+            ProgramOptions(debug=True, numba_debug=True),
+            marks=pytest.mark.skipif(
+                (_get_nvrtc_version_for_tests() or 0) < 13200,
+                reason="numba_debug requires libNVVM >= 13.2",
```

Comment:
The `skipIf` checks the NVRTC version when the reason given is the NVVM version. Is this an error, or is it because there's no way to get the NVVM version and the NVRTC and NVVM versions are assumed to be consistent?

Reply:
I just forgot to change it. Originally for internal testing I was using _get_libnvvm_version_for_tests check and extending it for 13.2, but then changed to NVRTC check. Which I think should be better for this particular case?